### PR TITLE
Update BUMP yamls

### LINKED ISSUE #182: Update BUMP yamls
See https://github.com/JCSDA-internal/mpas-jedi/pull/822 and https://github.com/JCSDA-internal/saber/pull/424.

### PR DESCRIPTION
**Pull request by byoung-joo**
### Description
* Companion to https://github.com/JCSDA-internal/mpas-jedi/pull/822.  YAML formats are changed for `multivariate strategy` key of BUMP.  No change in the result is expected.
* Build is updated to 25JAN2023.
* StaticB files are regenerated, due to a change in the group name formatting in some BUMP NetCDF files.
We can still use the previous versions of BUMP localization files.

### Issue closed

Closes #182 

### Tests completed
 - [x] 3dvar_OIE120km_WarmStart
 - [x] 3denvar_OIE120km_WarmStart
 - [x] 3dvar_OIE120km_ColdStart
 - [x] 3dvar_O30kmIE60km_ColdStart
 - [x] 3denvar_O30kmIE60km_WarmStart
 - [x] eda_OIE120km_WarmStart

### CODE REVIEW

**Comment by byoung-joo** [comment]
Six tests went through without an issue on Cheyenne. I have compared several results with previous version of MPAS-Workflow, but I don't have the results for all scenarios. The test results are located in `/glade/scratch/bjung/pandac/*_TEST.25JAN2023`.

Also, is the configuration `config/jedi/applications/3denvar-crossed.yaml` used in MPAS-Workflow ? 
I just changed the `multivariate strategy` key, but this was not tested and might not work properly...

**Comment by jjguerrette** [comment]
> Six tests went through without an issue on Cheyenne. I have compared several results with previous version of MPAS-Workflow, but I don't have the results for all scenarios. The test results are located in `/glade/scratch/bjung/pandac/*_TEST.25JAN2023`.

As long as the tests completed, as in the cylc suites complete and exit successfully, that is enough.  I do not check the results, because we do not have references.  More work to be done there.

> Also, is the configuration `config/jedi/applications/3denvar-crossed.yaml` used in MPAS-Workflow ? I just changed the `multivariate strategy` key, but this was not tested and might not work properly...

That yaml is not used in the tests or by any example scenarios.  It was a placeholder for when we return to the all-sky work.  Whoever gets back to that work can open a new issue/PR to fix the yaml if needed.

File: config/jedi/applications/3denvar.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ cost function:
             data directory: {{bumpLocDir}}
             files prefix: {{bumpLocPrefix}}
           drivers:
-            multivariate strategy: common
+            multivariate strategy: duplicated
```
Comment (jjguerrette) [inline config/jedi/applications/3denvar.yaml:58]: Does `common` need to be changed to `duplicated` in the localization generation too?  Asking so I can update my own offline yamls.

File: scenarios/base/variational.yaml [hunk 2]
```diff
@@ -247,13 +247,13 @@ variational:
       bumpCovStdDevFile: None
       bumpCovVBalDir: None
     60km:
-      bumpCovDir: /glade/p/mmm/parc/bjung/pandac_common/staticB/20230106_prebuilt/60km.NICAS_00
-      bumpCovStdDevFile: /glade/p/mmm/parc/bjung/pandac_common/staticB/20230106_prebuilt/60km.CMAT_00/mpas.stddev_0p33.2018-04-15_00.00.00.nc
-      bumpCovVBalDir: /glade/p/mmm/parc/bjung/pandac_common/staticB/20230106_prebuilt/60km.VBAL_00
+      bumpCovDir: /glade/p/mmm/parc/bjung/pandac_common/staticB/20230125_prebuilt/60km.NICAS_00
```
Comment (jjguerrette) [inline scenarios/base/variational.yaml:250]: Should I update localization files too?

**Review by jjguerrette — COMMENTED** [review]
Thank you BJ. Just a couple questions.

File: config/jedi/applications/3denvar.yaml (hunk 1 above)
Comment (byoung-joo) [reply config/jedi/applications/3denvar.yaml:58]: Yes, it should be updated there!

File: scenarios/base/variational.yaml (hunk 2 above)
Comment (byoung-joo) [reply scenarios/base/variational.yaml:250]: There is no change in the localization files (specifically for group name `common`), so we can use the existing ones.

Comment (byoung-joo) [reply scenarios/base/variational.yaml:250]: It is a bit complex... the `multivariate strategy` key is changed from `common` to `duplicated`, BUT the NetCDF group name `common` stays the same as before in BUMP NetCDF file.

Comment (jjguerrette) [reply scenarios/base/variational.yaml:250]: OK, then I will not update them.  Do the yaml settings below look correct to you for generating localization?

```yaml
  drivers:
    ## always on
    compute nicas: true
    write local nicas: true
    write global nicas: true
    write nicas grids: true

    ## same length scales for all variables
    multivariate strategy: duplicated
  nicas:
    resolution: 8.0
    explicit length-scales: true
    ## duplicated
    horizontal length-scale:
      - variables: [duplicated]
        value: HLocLengthKMe3
    vertical length-scale:
      - variables: [duplicated]
        value: VLocLengthKMe3
```

**Comment by byoung-joo** [comment]
> As long as the tests completed, as in the cylc suites complete and exit successfully, that is enough. I do not check the results, because we do not have references. More work to be done there.

Yes, all cylc suites complete and exit successfully!

> That yaml is not used in the tests or by any example scenarios. It was a placeholder for when we return to the all-sky work. Whoever gets back to that work can open a new issue/PR to fix the yaml if needed.

Thank you for your explanation, @jjguerrette. We can continue this part latertime! perhaps when we update the build version for benchmark experiment!

File: scenarios/base/variational.yaml (hunk 2 above)
Comment (byoung-joo) [reply scenarios/base/variational.yaml:250]: Let me double check this !

Comment (jjguerrette) [reply scenarios/base/variational.yaml:250]: I'm checking right now.  Will report back.

**Review by jjguerrette — APPROVED** [review]

File: scenarios/base/variational.yaml (hunk 2 above)
Comment (jjguerrette) [reply scenarios/base/variational.yaml:250]: I changed `variables: [duplicated]` to `groups: [common]`.  That appears to be working.

**Pull request merged**